.changeset/neat-llamas-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+"changelog": patch
```

Comment:
nit: changelogs are for user visible changes only. It is what we communicate to users about fixes/features/breaking changes. I suggest not including entries about our infra or internal setup, as it would add noise to people using the changelog to figure out what changed in the public API/product.

Reply:
I'm still figuring out the public boundary because of the two-step build process and what is considered public - you're right that the @generated annotation should only concern our infra, so I'll back this change out

Reply:
Removed the entry in 6b3d278c9e687a9abf90d7c3de02fb88535f247e